LIPs/LIP-25.md

```diff
@@ -8,9 +8,14 @@ created: 2020-06-09
 discussions-to: https://github.com/livepeer/LIPs/issues/25
 ---
 
-## Abstract
+## Introduction
```

Comment:
Let's keep the "Abstract" title in accordance with the [LIP-X](https://github.com/livepeer/LIPs/blob/master/LIP-X.md) template.

Reply:
fixed in 7cb8845